core/modules/file/file.module

```diff
@@ -1744,7 +1745,7 @@ function file_managed_file_browser_open($form, $form_state) {
 
   $commands = array();
   $commands[] = ajax_command_settings($settings, TRUE);
-  $commands[] = ajax_command_open_modal_dialog($title, $html, $options);
+  $commands[] = ajax_command_open_dialog('#backdrop-browser-modal', $title, $html, $options);
```

Comment:
The other selectors in this function are labeled `file-browser-*`. Could we change this to `#file-browser-modal`?

Reply:
Done